src/utils/alert.ts

```diff
@@ -128,10 +134,10 @@ export const sendAlerts = async ({
   const ipAddress = getIp()
   const message = getMessageForAlert(validation.alert, url, ipAddress, status)
   const sent = await Promise.all<any>(
-    notifications.map((notification) => {
+    notifications.map(async (notification) => {
```

Comment:
No need to use async anymore here because the `notifications` is mapped to array of Promises and then the Promise.all will wait for them. 

So no need to add `await` on each cases below as well.

Reply:
fixed